src/analysis/normed_space/pi_Lp.lean

```diff
@@ -317,4 +317,51 @@ variables {𝕜 p α}
 @[simp] lemma smul_apply : (c • x) i = c • x i := rfl
 @[simp] lemma neg_apply : (-x) i = - (x i) := rfl
 
+variables {ι' : Type*}
+variables [fintype ι']
+
+variables {E : Type*} [normed_group E] [normed_space 𝕜 E]
+
+/-- An equivalence of finite domains induces a linearly isometric equivalence of finitely supported
+functions-/
+def _root_.linear_isometry_equiv.Pi_congr_left (e : ι ≃ ι') :
```

Comment:
I think the name should be something like `linear_isometry_equiv.pi_Lp_congr_left`, because this is a nonstandard norm structure (the Lp norm) on the pi-type -- the standard one (which, I guess, should also get a version of this construction someday) is the L-infinity norm.